### DISCUSSION TITLE
geo-osint-locating-kiffa-mauritania/

### DISCUSSION THREAD

**Discussion by giscus** [neospl0it/neospl0it.github.io#4] [Comments]
# geo-osint-locating-kiffa-mauritania/

Explore how OSINT techniques can accurately pinpoint locations from minimal clues. A case study on geospatial tools and visual analysis to trace coordinates in Kiffa, Mauritania.

https://ne0day.github.io/geo-osint-locating-kiffa-mauritania/

**Comment by neospl0it** [comment]
Test {Nice Blog}
